### PR TITLE
Updating Maintainers

### PR DESCRIPTION
**Pull request by kekaichinose**
Signed-off-by: Keka Ichinose <rikoomi@hotmail.com>

## Description

Updating maintainers to reflect the current team members

## Related Issue

https://github.com/inspec/inspec/pull/4287

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by miah — APPROVED** [review]
Thanks @kekaichinose ! =)

File: MAINTAINERS.toml [hunk 1]
```diff
@@ -21,32 +21,31 @@ project lead.
...
+        lieutenant = "btm"
 
         maintainers = [
-          "chris-rock",
-          "adamleff",
-          "alexpop",
-          "jquick"
+          "clintoncwolfe",
+          "miah",
+          "zenspyder"
```
Comment (clintoncwolfe) [inline MAINTAINERS.toml]: Ryan's username is zenspider, not zenspyder

File: MAINTAINERS.toml [hunk 2]
```diff
@@ -21,32 +21,31 @@ project lead.
...
+  [people.miah]
+    Name = "Miah Johnson"
+    GitHub = "miah"
 
-  [people.alexpop]
-    Name = "Alex Pop"
-    GitHub = "alexpop"
+  [people.zenspyder]
+    Name = "Ryan Davis"
+    GitHub = "zenspyder"
```
Comment (clintoncwolfe) [inline MAINTAINERS.toml]: Ryan's username correction here, too

**Review by clintoncwolfe — CHANGES_REQUESTED** [review]
Looks like Ryan's username is wrong otherwise looks good!

**Review by clintoncwolfe — APPROVED** [review]
Thanks, looks great!

**Pull request merged**